11ty/content-warning-guidelines/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Content Warning Guidelines
+---
+
+There are a wide variety of topics that can cause unnecessary harm to people, inducing negative feelings, anxiety, or trauma. The goal of Content Warnings or Trigger Warnings is to allow the audience to prepare themselves, or if they choose to, avoid the topic completely.
+
+## What is the difference between a Content Warning and a Trigger Warning
+
+You may well have heard these two different phrases used to warn of different topics, but what is the differences? It is sometimes said that content warnings are used to help people avoid topics that may cause offence, upset or anxiety, and that trigger warnings are used when warn of a topic that may cause a traumatic reaction.
+
+This however is very hard to gauge. People can have very different reactions, just as they may have very different personal experiences, with any topic that you may discuss. Their form follows the same rules though and so for this reason we will be using the term Content Warning for these guidelines.
+
+The most important thing though, is that a warning is given, rather than which term is used.
+
+## When to give a Content Warning
+
+Any time that you discuss a topic that you believe may cause a negative reaction for anyone, at any scale, then it would be best to give a content warning. It is best to try and think objectively about this, rather then just through the spectrum of your own personal experience and triggers.
+
+To help with this, definitions on Self Defined will advise you when we recommend that you give a content warning. If you are ever in doubt though, we would recommend to side on deciding to give a warning rather than not.
+
+## How to give a Content Warning
+
+There are two standardised ways of giving a content warning in writing:
+
+<ul class="list-default">
+  <li>Content Warning: [topic or topics]</li>
+  <li>CW: [topic or topics]</li>
+</ul>
+
+We would recommend that wherever possible you choose the first full text example. This will ensure that people who may not be familiar with the acronym of CW will still fully understand. In the case of media such as video, it is vital that it is given in a way that everyone will be able to access it, including through audio or visual only.
```

Comment:
```suggestion
We would recommend that wherever possible you choose the first full text example. This will ensure that people who may not be familiar with the acronym of CW will still fully understand. In the case of media such as video, it is vital that it is given in a way that everyone will be able to access it, including through audio and visual cues.
```